src/extensions/yfm/YfmTabs/utils.ts

```diff
@@ -1,6 +1,12 @@
-import {EditorView} from 'prosemirror-view';
+import type {EditorView} from 'prosemirror-view';
 
-import {tabPanelType} from '.';
+import {tabPanelType} from './const';
+
+export const rAF = (fn: () => void) => {
+    requestAnimationFrame(() => {
+        requestAnimationFrame(fn);
+    });
+};
```

Comment:
To improve code readability and maintainability, it is recommended to add comments explaining the necessity of the double `requestAnimationFrame` call and its use cases, such as the following: “The double requestAnimationFrame call ensures that all previous DOM updates are completed, and the browser has had time to perform necessary operations, such as style recalculations and layout adjustments, before executing the provided function.”

Reply:
The name `rAF` does not reflect its specific purpose. A more descriptive name, such as `runAfterNextPaint` or `executeAfterRender`, would improve code clarity.

Reply:
fixed